remotespark/datawidgets/plotlygraphs/graphrenderer.py

```diff
@@ -13,15 +13,10 @@
 
 class GraphRenderer(object):
 
-    already_initialized = False
-
-    def __init__(self, testing=False):
-        if not testing and not GraphRenderer.already_initialized:
-            init_notebook_mode()
-            GraphRenderer.already_initialized = True
-
     @staticmethod
     def render(df, encoding, output):
+        with output:
```

Comment:
The previous solution wrote to file while this one writes to ouput (which is a widget). Hence, notebook file size does not increase.